xml/rmt_overview.xml

```diff
@@ -61,18 +56,6 @@
      or internal repositories into your update solution.
```

Comment:
@svenseeberg That sentence seems weird: integrate additional software **OR** update external
     or internal repositories into your update solution?

Reply:
Indeed. Let's change it.